helm/charts/determined/templates/change-password.yaml

```diff
@@ -22,17 +22,21 @@ spec:
       restartPolicy: OnFailure
       containers:
       - name: change-password
-        image: "{{ .Values.imageRegistry }}/utility:py-3.7-pw-changer"
+        image: "{{ .Values.imageRegistry }}/environments:py-3.8-pytorch-1.12-tf-2.8-cpu-9d07809"
```

Comment:
issue: could we please do the same "split" action as [here](https://github.com/determined-ai/determined/blob/master/helm/charts/determined/templates/master-config.yaml#L134) to preserve the image name?

[bumpenvs](https://github.com/determined-ai/determined/tree/master/tools/scripts) scripts which update the environment image name to the current version effectively do "find & replace" on the full image name. if it's literal `determinedai/environments:py-3.8-pytorch-1.12-tf-2.8-cpu-9d07809` string, it'd be updated okay, otherwise it'd be missed, and we'd have an outdated image version in the code.

Reply:
resolved in new commit -- I wasn't sure where to store the $cpuImage variable, so happy to move it to another place in the .yaml file if that makes it more readable